solr/solr-ref-guide/src/nested-documents.adoc

```diff
@@ -0,0 +1,321 @@
+
+=== JSON Examples
+
+This example is equivalent to the XML example above.
+Again, the field labelled relationship is preferred.
+The labelled relationship here is one child document but could have been wrapped in array brackets.
+For the anonymous relationship, note the special `\_childDocuments_` key whose contents must be an array of child documents.
+
+[source,json]
+----
+[
+  {
+    "id": "1",
+    "title": "Solr adds block join support",
+    "content_type": "parentDocument",
+    "comments": [{
+        "id": "2",
+        "content": "SolrCloud supports it too!"
+      },
+      {
+        "id": "3",
+        "content": "New filter syntax"
+      }
+    ]
+  },
+  {
+    "id": "4",
+    "title": "New Lucene and Solr release is out",
+    "content_type": "parentDocument",
+    "_childDocuments_": [
+      {
+        "id": "5",
+        "comments": "Lots of new features"
+      }
+    ]
+  }
+]
+----
+
+.Root Only Mode
+[NOTE]
+====
+ In Root-only schemas, these two documents will result in the same docs being indexed(Root-only schemas do not honor nested relationships).
+ When quried, child docs will be appended to _childDocuments_ key.
+====
+
+== Searching Nested Documents
+
+ * `<<nested-documents.adoc#child-doc-transformer,[child]>>` Document Transformer
+ * <<nested-documents.adoc#children-query-parser,Block Join Children Query Parser>>
+ * <<nested-documents.adoc#parents-query-parser,Block Join Parent Query Parser>>
+ * <<nested-documents.adoc#combining-block-join-query-parsers-with-child-doc-transformer,Advanced Queries>>
+ * <<blockjoin-faceting#blockjoin-faceting, Faceting using nested documents(blockChildren and blockParents)>>
+
+=== Query Examples
+
+For the upcoming examples, assume the following documents have been indexed:
+
+====
+[source,json]
+----
+[
+  {
+    "id": "1",
+    "title": "Cooking Recommendations",
+    "tags": ["cooking", "meetup"],
+    "posts": [{
+        "id": "2",
+        "title": "Cookies",
+        "comments": [{
+            "id": "3",
+            "content": "Lovely recipe"
+          },
+          {
+            "id": "4",
+            "content": "A-"
+          }
+        ]
+      },
+      {
+        "id": "5",
+        "title": "Cakes"
+      }
+    ]
+  },
+  {
+    "id": "6",
+    "title": "For Hire",
+    "tags": ["professional", "jobs"],
+    "posts": [{
+        "id": "7",
+        "title": "Search Engineer",
+        "comments": [{
+           "id": "8",
+           "content": "I am interested"
+         },
+         {
+           "id": "9",
+           "content": "How large is the team?"
+         }
+        ]
+      },
+      {
+        "id": "10",
+        "title": "Low level Engineer"
+      }
+    ]
+  }
+]
+----
+====
+
+==== Child Doc Transformer
+
+Can be used enrich query results with the documents' descendants. +
+For a detailed explanation of this parser, click <<transforming-result-documents.adoc#child-childdoctransformerfactory, here>>.
+
+* `q=id:1,
```

Comment:
Why is this a bullet? Usually bullets should only be used when there is a list, but there is only one item here. 

Also, this is an example? I'm not sure what I'm looking at. It should have a bit of context, "If we have a query `q=id:1 ...`"